modules/fedimint-dummy-tests/tests/tests.rs

```diff
@@ -204,10 +210,15 @@ mod fedimint_migration_tests {
         dbtx.commit_tx().await;
     }
 
+    fn create_client_states() -> (Vec<DummyStateMachine>, Vec<DummyStateMachine>) {
```

Comment:
This should add a few states that won't be migrated so that we verify that we don't accidentally delete any states